src/main/java/seedu/taassist/model/session/SessionData.java

```diff
@@ -0,0 +1,57 @@
+package seedu.taassist.model.session;
+
+import static seedu.taassist.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.taassist.model.uniquelist.Identity;
+
+/**
+ * Represents a session data of student in a module.
+ */
+public class SessionData implements Identity<SessionData> {
+
+    private final Session session;
+    private final double grade;
+
+    /**
+     * Constructs a {@code SessionData} with the given session and student session data.
```

Comment:
```suggestion
     * Constructs a {@code SessionData} with the given session and grade.
```
Mentioning `student` here feels unnecessary since there is no student object or information present in the class.

Reply:
Yes .. there was before. :joy: